implementations/JavaScript_jsonpath/package-lock.json

```diff
@@ -14,11 +14,11 @@
       "resolved": "https://registry.npmjs.org/escodegen/-/escodegen-1.11.1.tgz",
       "integrity": "sha512-JwiqFD9KdGVVpeuRa68yU3zZnBEOcPs0nKW7wZzXky8Z7tffdYUHbe11bPCV5jYlK6DVdKLWLm0f5I/QlL0Kmw==",
       "requires": {
-        "esprima": "^3.1.3",
```

Comment:
Not sure why npm behaves so differently across OS X and Linux, but it's the same on my machine. Happy to ignore for now.